src/styles/blocks/card.scss

```diff
@@ -0,0 +1,95 @@
+body {
+  margin: 0;
+}
+
+.container {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 200px;
+  height: 408px;
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  font-family: $main-font-family;
+  color: $main-color;
+
+  &__photo {
+    background-image: url(../images/imac.jpeg);
+    width: 100%;
+    height: 134px;
+    background-size: contain;
+    background-repeat: no-repeat;
+    margin: 0 3px 40px 2px;
+  }
```

Comment:
Instead of using a background image for the product photo, consider using an 'img' tag. This is better for accessibility and SEO. Also, it will allow you to use alt text to describe the image.